endpoints/openrtb2/auction_test.go

```diff
@@ -6108,3 +6109,815 @@ func TestValidateAliases(t *testing.T) {
 func fakeNormalizeBidderName(name string) (openrtb_ext.BidderName, bool) {
 	return openrtb_ext.BidderName(strings.ToLower(name)), true
 }
+
+func Test_setSecBrowsingTopcisImplicitly(t *testing.T) {
```

Comment:
The convention we use for test names is "Test" + name of function starting with a capital letter. This should be `TestSetSecBrowsingTopcisImplicitly`.

Reply:
done